### PR TITLE
Added download size to main download button

### PR DESCRIPTION
**Pull request by taskinoz**
Pulling the data from the Github API to show the size of the download on the main download button.

image

### CODE REVIEW

File: web/script/index.js [hunk 1]
```diff
@@ -2,7 +2,10 @@ fetch('https://api.github.com/repos/R2Northstar/Northstar/releases/latest')
     .then(response => response.json())
     .then(data => {
         document.querySelectorAll('.download').forEach(d => { d.href = data.assets[0].browser_download_url });
-        console.log("Successfully fetched latest release download URL. Updating download links.")
+        console.log("Successfully fetched latest release download URL. Updating download links.");
+        var filesize = data.assets[0].size;
+        var sizeinmb = filesize/1024/1024;
+        document.querySelectorAll('.big-button.download > span').forEach(e => { e.innerHTML = 'DOWNLOAD ('+sizeinmb.toFixed(1)+' MB)'});
```
Comment (barnabwhy) [inline web/script/index.js:8]: innerText is safer so should be used even though we trust github to not tell us the file size is some arbitrary html string

File: web/script/index.js [hunk 2]
```diff
@@ -2,7 +2,10 @@ fetch('https://api.github.com/repos/R2Northstar/Northstar/releases/latest')
     .then(response => response.json())
     .then(data => {
         document.querySelectorAll('.download').forEach(d => { d.href = data.assets[0].browser_download_url });
-        console.log("Successfully fetched latest release download URL. Updating download links.")
+        console.log("Successfully fetched latest release download URL. Updating download links.");
+        var filesize = data.assets[0].size;
```
Comment (barnabwhy) [inline web/script/index.js:6]: i suggest using camelCase to make variable names a bit more readable

**Review by barnabwhy — CHANGES_REQUESTED** [review]

File: web/script/index.js (hunk 1 above)
Comment (taskinoz) [reply web/script/index.js:8]: I forgot about that, I just copied what I had from discord last night.

Comment (taskinoz) [reply web/script/index.js]: Changed to innerText

File: web/script/index.js (hunk 2 above)
Comment (taskinoz) [reply web/script/index.js]: Renamed both variables with camelCase

**Review by abarichello — APPROVED** [review]

**Pull request merged**